src/app/[locale]/dash/logout/page.tsx

```diff
@@ -0,0 +1,14 @@
+"use client";
+
+import { useClerk } from "@clerk/nextjs";
+import { useEffect } from "react";
+
+export default function LogOutPage() {
+  const { signOut } = useClerk();
+
+  useEffect(() => {
+    signOut(() => window.location.replace("/"));
+  }, [signOut]);
+
+  return <span>Logging out...</span>;
```

Comment:
add some styling in this

Reply:
why? it wont appear for even 1ms, its fallback while user is being logged off